get-started/versioning-availability.md

```diff
@@ -7,208 +7,53 @@ products:
 description: Learn how Elastic handles versioning and feature availability in the docs. Find the product versions that are supported, how to read availability badges, and...
 ---
 
-# Versioning and availability in Elastic Docs
+# Versioning and availability
 
-Learn how Elastic Docs handles versioning, feature availability, and how to find the right documentation for your deployment type and product version. Find answers to common questions about the Elastic Stack versioning and confidently navigate our continuously updated documentation.
+Learn how Elastic products are versioned, the lifecycle of features, and how to find the right documentation for your deployment type and product version. Find answers to common questions about the versioning and confidently navigate our continuously updated documentation.
 
-## Frequently asked questions
+## Understanding versioning
 
-### Where can I find documentation for the latest version of the {{stack}}?
+Each deployment type has a unique versioning strategy. For example, Serverless and ECH have releases that are distinguished by their date. Because Serverless is automatically updated, you always have the latest updates.
 
-You’re in the right place! All documentation for Elastic Stack 9.0.0 and later is available at [elastic.co/docs](https://www.elastic.co/docs), including the latest {{version.stack| M.M}} version and any future versions in the 9.x series.
-
-Need docs for an earlier version? Go to [elastic.co/guide](https://www.elastic.co/guide).
-
-### Why doesn't Elastic have separate documentation for each version?
-
-Starting with {{stack}} 9.0.0, Elastic no longer publishes separate documentation sets for each minor release. Instead, all changes in the 9.x series are included in a single, continuously updated documentation set.
-
-This approach helps:
-* Reduce duplicate pages
-* Show the full history and context of a feature
-* Simplify search and navigation
-
-### How do I know content was added in a specific version?
```

Comment:
there was a lot of concern that our badges were not understandable. are we ready to remove this guidance? did you check with the person who added it?